uaclient/entitlements/fips.py

```diff
@@ -535,8 +547,16 @@ def messaging(self) -> MessagingOperationsDict:
 
     def _perform_enable(self, silent: bool = False) -> bool:
         if super()._perform_enable(silent=silent):
-            self.cfg.notice_file.try_remove(
-                "", messages.FIPS_DISABLE_REBOOT_REQUIRED
+            notices.remove(
+                self.cfg.root_mode,
+                NoticesDetails.FIPS_DISABLE_REBOOT_REQUIRED.value,
+            )
+            services_once_enabled = (
+                self.cfg.read_cache("services-once-enabled") or {}
+            )
+            services_once_enabled.update({self.name: True})
+            self.cfg.write_cache(
+                key="services-once-enabled", content=services_once_enabled
```

Comment:
I think this must've accidentally slipped in during a rebase - it can be removed